chart/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
I'm not particularly familiar with Helm versioning, but should this match up with our RBAC Manager releases? I'm assuming this would go live in `v0.2.1`.

Reply:
Good catch. I actually think we should do 0.3.0 since this is a little more break-y so I went ahead and planned for that.